xml/deployment_cifs.xml

```diff
@@ -156,13 +175,10 @@
 &prompt.root;smbpasswd -a <replaceable>USERNAME</replaceable>
 </screen>
       <para>
-       Since <systemitem>vfs_ceph</systemitem> does not require a file system
-       mount, the share path is interpreted as an absolute path within the
-       &ceph; file system on the attached &ceph; cluster. For successful share
-       I/O, the path's access control list (ACL) needs to permit access from
-       the mapped user for the given Samba client. You can modify the ACL by
-       temporarily mounting via the &cephfs; kernel client and using the
-       <command>chmod</command>, <command>chown</command>, or
+       For successful I/O, the share path's access control list (ACL) needs to
+       permit access from the mapped user for the given &samba; client. You can
```

Comment:
This might be my unfamiliarity with Ceph, but I do not understand the phrase

> the mapped user for the given Samba client

How can a user be mapped? I would think that a user is the entity **doing** the mapping to a share, not the entity **being mapped**. I may be wrong.

If I am right, I would suggest:

> permit access by the user mapping to the given Samba client

Reply:
Good point, use of `map` here is a little confusing. It's a bit of an overloaded term, as it can refer to mapping of a network drive on Windows, etc.
The mapping behaviour that I'm trying to describe is performed by Samba, where it translates between SMB user/group identities (name+SID) and UNIX user/group identities.
I'll have a think about how this can be reworded to be a little clearer. Thanks for your feedback!


, as a reader may think it relates to mapping a